org.kde.skanpage.json

```diff
@@ -0,0 +1,262 @@
+                }
+            ]
+        },
+        {
+            "name": "sane-airscan",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/alexpevzner/sane-airscan",
+                    "tag": "0.99.27",
+                    "commit": "053f5654cd4d16f077fe5cb40d32004aaa321462"
+                }
+            ]
+        },
+        {
+            "name": "ksanecore",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/release-service/23.08.4/src/ksanecore-23.08.4.tar.xz",
+                    "sha256": "57d116917278369d9468c6a86d308019a76c1c4080b110b6cf9b589600d500e4",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 8763,
+                        "stable-only": true,
+                        "url-template": "https://download.kde.org/stable/release-service/$version/src/ksanecore-$version.tar.xz"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "leptonica",
+            "rm-configure": true,
+            "config-opts": [
+                "--disable-static"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/DanBloomberg/leptonica/releases/download/1.83.1/leptonica-1.83.1.tar.gz",
+                    "sha256": "8f18615e0743af7df7f50985c730dfcf0c93548073d1f56621e4156a8b54d3dd",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 1549,
+                        "stable-only": true,
+                        "url-template": "https://github.com/DanBloomberg/leptonica/releases/download/$version/leptonica-$version.tar.gz"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "tesseract",
+            "config-opts": [
+                "--disable-static"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/tesseract-ocr/tesseract/archive/refs/tags/5.3.3.tar.gz",
+                    "sha256": "dc4329f85f41191b2d813b71b528ba6047745813474e583ccce8795ff2ff5681",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 4954,
+                        "stable-only": true,
+                        "url-template": "https://github.com/tesseract-ocr/tesseract/archive/refs/tags/$version.tar.gz"
+                    }
+                },
+                {
+                    "type": "file",
+                    "url": "https://github.com/tesseract-ocr/tessdata_best/raw/4.1.0/eng.traineddata",
```

Comment:
There's a gazillion more languages supported by tesseract, what was your rationale for including this exact subset?

Reply:
https://github.com/flathub/org.kde.skanpage/issues/1